src/mapper_class.cpp

```diff
@@ -323,6 +356,58 @@ void MapperClass::PublishRadiusMarkers(const Eigen::Vector3d &center,
     obstacle_radius_marker_pub_.publish(obstacle_radius_marker);
 }
 
+void MapperClass::PublishPathPlanningConfigMarkers(const pensa_msgs::PathPlanningConfig &path_planning_config,
+                                                   const std::string &inertial_frame_id) {
+    const std::string ns = "path_planning_config";
+    const double thickness = 0.1;
+    std_msgs::ColorRGBA no_fly_zone_color = visualization_functions::Color::Red();
+    std_msgs::ColorRGBA fly_zone_color = visualization_functions::Color::Green();
+    no_fly_zone_color.a = 0.3;  // make them slightly transparent
+    fly_zone_color.a = 0.15;    // make them slightly transparent
+    visualization_msgs::MarkerArray no_fly_zones_markers;
+    visualization_functions::DrawPathPlanningConfig(path_planning_config, ns, inertial_frame_id, no_fly_zone_color,
+                                                    fly_zone_color, thickness, &no_fly_zones_markers);
+    path_planning_config_pub_.publish(no_fly_zones_markers);
+}
+
+void MapperClass::PublishPathPlanningPathMarkers(const std::vector<Eigen::Vector3d> &path,
```

Comment:
function that publishes A* trajectory into Rviz. Notice that it publishes two trajectories: the full A* trajectory, and another one that has been compressed to reduce the number of intermediate waypoints